de.dlr.sc.virsat.swtbot.test/src/de/dlr/sc/virsat/swtbot/test/RoleManagementTest.java

```diff
@@ -158,8 +176,17 @@ public void testApplyDisciplineRecursive() {
 		// and one for a special ED.
 		// Both are initially owned by the current user.
 		createNewDiscipline("Domain_One", null);
-		createNewDiscipline("Domain_Two", "other_user");
-		createNewDiscipline("Domain_Three", "third_user");
+		createNewDiscipline("Domain_Two", Collections.singletonList("user2")).click(1);
+		bot.list().select(0);

```

Comment:
Ah, it will automatically add the default user to the discipline? That's why you have to remove the first user here? Then I think we should rather update the createNewDiscipline to rename the initial user to the first name from the list... What do you think?

Reply:
The problem is, it is still complex to get the default user.